docker/ceph/centos/Dockerfile

```
@@ -1,21 +1,30 @@
-ARG CENTOS_VERSION=8
+ARG CENTOS_VERSION=${CENTOS_VERSION:-8}
 FROM quay.io/centos/centos:stream$CENTOS_VERSION as ceph-base
 ARG CENTOS_VERSION
 
-RUN dnf --disablerepo '*' --enablerepo=extras swap centos-linux-repos centos-stream-repos -y
+RUN if [ "$CENTOS_VERSION" = "8" ]; then \
+        dnf --disablerepo '*' --enablerepo=extras centos-linux-repos centos-stream-repos swap -y; \
```

This is the reason of the current failure:
```suggestion
        dnf --disablerepo '*' --enablerepo="extras centos-linux-repos centos-stream-repos" swap -y; \
```

---

thank you @epuertat I might have forgot to update but I removed centos 8 completely and went with centos9. centos8 was failing to pull in its dependencies for appstreams and others...